client/packages/android/package.json

```diff
@@ -4,6 +4,8 @@
   "private": true,
   "devDependencies": {
     "@capacitor-community/barcode-scanner": "^3.0.3",
+    "@capacitor-community/keep-awake": "^3.0.0",
+    "@capacitor/app": "^4.1.1",
```

Comment:
Is it correct that they are under devDependencies? if yes why?

Reply:
It's a bit strange isn't it. They don't need to be dependencies because capacitor is including the java files (`npx cap sync` and then the build process) and that's all that is required at runtime.

I've included them in the android/package just so that `npx cap sync` will detect the plugins correctly. Apart from that, the packages are used only for typescript checking, so a dev dep is sufficient.

I'll move them to runtime dependencies in the android package and will keep them as dev dependencies in the other packages. I think that's clearer

Reply:
Ok thanks!